src/components/Child.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import Calculator from './Calculator';
+
+function Child() {
+  return (
+    <section>
+      <div id="calculator-contaienr">
+        <div id="display">0</div>
+        <Calculator />
+      </div>
+    </section>
+  );
+}
```

Comment:
- [ ] Great work so far! However, as part of the project requirements, the 'Child' component needs to be nested inside the 'Calculator' component, not the other way around. Please make sure to adjust the component structure accordingly. Once this is done, you can import the 'Calculator' component into the top-level 'App' component. Thanks!